content/blog/kallisti5/2021-03-22_feb_2021_survey_results.md

```diff
@@ -0,0 +1,138 @@
++++
+type = "blog"
+title = "Feb 2021 - Survey Results"
+author = "kallisti5"
+date = "2021-03-22 17:37:59-05:00"
+tags = ["haiku", "software", "survey"]
++++
+
+Earier this month, I created a quick survey to guage what topics are most important to the Haiku communtity.
+We had 387 responses making this pretty substantial in helping us evaluate places we need to improve.<br/><br/>
+Thanks to everyone who filled out this survey!<br/>
+I'll be going over the results and adding a little commentary along the way. Lets go!
+
+## Which version of Haiku do you normally use?
+
+<img src="/files/survey_feb2021/1.png" alt="59.7% Stable, 40.3% Nightly"><br/>
+
+So far, not too suprising. A lot of people value the stable R1/Beta 2 release, but most prefer the nightly images. This makes sense as R1 / Beta 2 was released around 9 months ago.
+
+## What architecture do you normally use?
+
+<img src="/files/survey_feb2021/2.png" alt="85.5% x86_64 (64-bit), 14.5% x86_gcc2 (32-bit)"><br/>
+
+We continue to support the 32-bit version of Haiku for its BeOS binary compatibility. However, it appears the trend continues to be users gravitating to the more modern x86_64.
+This further solidifies the idea that after R1, the 32-bit version of Haiku will be eventually deprecated.
+
+## How often do you use haiku on a virtual machine vs physical hardware?
+
+<img src="/files/survey_feb2021/3.png" alt="41.3% of people never use Haiku in a VM. 32.3% of people never use haiku on physical hardware"><br/>
+
+This one is interesting. It seems more users prefer running Haiku on physical hardware. Those who run Haiku on physical hardware also run Haiku daily more often then users who run Haiku in a virtual machine.
+
+## What is the most "common problem" you see when using Haiku?
+
+<img src="/files/survey_feb2021/4.png" alt="53.7% Hardware/Driver support, 37.7% Lack of software"><br/>
+
+This one doesn't give a lot of interesting data. Hardware / Driver support in the top position isn't suprising. Haiku supports a large amount of hardware natively for our size... however we definitely stagger behind Linux and the BSD's on hardware support.
+
+## What class of hardware has the most problems in your experience
+
+<img src="/files/survey_feb2021/5.png" alt="45.5% Graphics, 19.6% Network, "><br/>
+
+Not suprising.  While Haiku supports quite a few mainstream graphics cards, we are always finding edge cases and broken modesetting.
+There are some long-planned projects trying to piggyback on Linux's graphics drivers... however nothing has even really taken shape beyond our native drivers.
+
+Network is actually a little suprising to me here. I personally don't see too many issues with network cards.
+I'm wondering if by 'Network' users actually mean our control panel / wifi network selection / etc?
```

Comment:
That seems like a wild guess. There are many wifi chipstes that aren't supported (I have some affected machines myself), and even for those which work, things aren't always super stable.

In general the state of Wifi is not so great, not to mention the lack of usb wifi solutions (which would at least be easy to add to existing hardware as a stopgap)

Reply:
It is a guess to be honest.

My own "wired" experiences have been perfect for quite a while now across a wide range of hardware... which is what led me to focusing on Wifi.

So.. instead of mentioning our network control panel... Wifi driver support (USB and PCI) and Wifi network selection?

I'm bringing up the wifi network selection because I know we already have a few issues there around not "merging similar AP names"      We need to put wpa_supplicant into a fork'd git repository tbh.  All of our wifi code exists within a haikuports patch.